org.eclipse.xtext.xbase.testlanguages/META-INF/MANIFEST.MF

```diff
@@ -1,48 +1,49 @@
- org.eclipse.xtext.xbase.testlanguages.bug462047.bug462047.util,
- org.eclipse.xtext.xbase.testlanguages.bug462047.serializer,
- org.eclipse.xtext.xbase.testlanguages.bug462047.parser.antlr,
- org.eclipse.xtext.xbase.testlanguages.bug462047.parser.antlr.internal,
- org.eclipse.xtext.xbase.testlanguages.bug462047.validation,
- org.eclipse.xtext.xbase.testlanguages.bug462047.jvmmodel
-
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Xbase Xtext Testlanguages for UI and Runtime - Runtime
+Bundle-Vendor: Eclipse Xtext
+Bundle-Version: 2.14.0.qualifier
+Bundle-SymbolicName: org.eclipse.xtext.xbase.testlanguages; singleton:=true
+Bundle-ActivationPolicy: lazy
+Require-Bundle: org.eclipse.xtext;visibility:=reexport,
+ org.eclipse.xtext.xbase;resolution:=optional;visibility:=reexport,
+ org.eclipse.xtext.generator;resolution:=optional,
+ org.apache.commons.logging;bundle-version="1.0.4";resolution:=optional,
+ org.eclipse.emf.codegen.ecore;bundle-version="2.10.2";resolution:=optional,
+ org.eclipse.emf.mwe.utils;resolution:=optional,
+ org.eclipse.emf.mwe2.launch;resolution:=optional,
+ org.eclipse.xtext.util,
+ org.eclipse.emf.ecore;bundle-version="2.10.2",
+ org.eclipse.emf.common;bundle-version="2.10.1",
+ org.antlr.runtime,
+ org.eclipse.xtext.common.types,
+ org.eclipse.xtext.xbase.lib;bundle-version="2.14.0",
+ org.objectweb.asm;bundle-version="[5.0.1,7.0.0)";resolution:=optional
+Import-Package: org.apache.log4j
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Export-Package: org.eclipse.xtext.xbase.testlanguages,
+ org.eclipse.xtext.xbase.testlanguages.services,
+ org.eclipse.xtext.xbase.testlanguages.xImportSectionTestLang,
+ org.eclipse.xtext.xbase.testlanguages.xImportSectionTestLang.impl,
+ org.eclipse.xtext.xbase.testlanguages.xImportSectionTestLang.util,
+ org.eclipse.xtext.xbase.testlanguages.serializer,
+ org.eclipse.xtext.xbase.testlanguages.parser.antlr,
+ org.eclipse.xtext.xbase.testlanguages.parser.antlr.internal,
+ org.eclipse.xtext.xbase.testlanguages.validation,
+ org.eclipse.xtext.xbase.testlanguages.formatting,
+ org.eclipse.xtext.xbase.testlanguages.jvmmodel,
+ org.eclipse.xtext.xbase.testlanguages.contentAssistFragmentTestLang,
+ org.eclipse.xtext.xbase.testlanguages.contentAssistFragmentTestLang.impl,
+ org.eclipse.xtext.xbase.testlanguages.contentAssistFragmentTestLang.util,
+ org.eclipse.xtext.xbase.testlanguages.bug462047,
+ org.eclipse.xtext.xbase.testlanguages.bug462047.services,
+ org.eclipse.xtext.xbase.testlanguages.bug462047.bug462047,
+ org.eclipse.xtext.xbase.testlanguages.bug462047.bug462047.impl,
+ org.eclipse.xtext.xbase.testlanguages.bug462047.bug462047.util,
+ org.eclipse.xtext.xbase.testlanguages.bug462047.serializer,
+ org.eclipse.xtext.xbase.testlanguages.bug462047.parser.antlr,
+ org.eclipse.xtext.xbase.testlanguages.bug462047.parser.antlr.internal,
+ org.eclipse.xtext.xbase.testlanguages.bug462047.validation,
+ org.eclipse.xtext.xbase.testlanguages.bug462047.jvmmodel
+
```

Comment:
remove empty line

Reply:
Removed the empty lines (there were two other files with empty lines).